src/score.h

```diff
@@ -261,16 +305,71 @@ struct ScoreFunction
                     logToConsole(log);
                     return false;
                 }
+
+                if (!allocatePool(maxDuration * sizeof(PoolSynapseData), (void**)&(cb._poolSynapseTickData)))
+                {
+                    CHAR16 log[256];
```

Comment:
Please declare `CHAR16 log[256];` only once, in a common scope. Memory on stack isn't reserved when the block is entered but at the beginning of the function. So probably all log buffers appearing in this function are reserved, leading to a large stack requirement.